src/ugo/ugo.F90

```diff
@@ -2189,7 +2189,7 @@ SUBROUTINE UGO_solve_direct( data, userdata, status, x, f, g, h, eval_FGH )
 
      INTERFACE
        SUBROUTINE eval_FGH( status, x, userdata, f, g, h )
-       USE GALAHAD_USERDATA_double
+       USE GALAHAD_USERDATA_precision
```

Comment:
Nick, this is the culprit of our issue of Friday! A `GALAHAD_USERDATA_double` was still hard coded in `ugo.F90`.
I suppose that you compile the double precision `.mod` before the single precision one on our computer.

Reply:
Indeed, poor old Harry (he plays for my club, and usually scores good penalties!). I would hate to take a penalty against my own team mate (Hugo Lloris also plays for us)